frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -61,6 +61,8 @@ export const BREAKPOINT_COLUMN_COUNTS: Record<DashboardLayoutSize, number> = { s
 export const MIN_ITEM_WIDTH_UNITS = 3
 export const MIN_ITEM_HEIGHT_UNITS = 5
 
+export const DASHBOARD_REFRESH_INTERVAL_MINUTES = 5
```

Comment:
Should we call this `DASHBOARD_MIN_MANUAL_REFRESH_INTERVAL_MINUTES`?

Reply:
yup that sounds better - I just copied the name from the insights codebase and prepended with dashboard